http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200719_add_app_custom_env_vars.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="gabriela" id="add_app_custom_env_vars">
+        <addColumn tableName="APP">
+            <column name="customEnvironmentVariables" type="varchar(2048)" >
```

Comment:
Not sure I understand? This `customEnvironmentVariables` is available to API clients and Terra UI

Reply:
by environment variables, we're talking about `WORKSPACE_NAME` AND `WORKSPACE_BUCKET` here right? or we're talking about actual client set envs?

Reply:
I think we'll use it to set `WORKSPACE_NAME` AND `WORKSPACE_BUCKET` but callers could set arbitrary EVs if they want

Reply:
Not sure I understand your first comment @Qi77Qi. I modeled this after how we set custom environment variables for runtimes, the UI can pass whichever var it needs to and like Rob said, we're passing them to the helm install command as key/value pairs

Reply:
https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala#L48 and https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/leonardoServiceContractModel.scala#L32 for reference

Reply:
I understand we're passing them to helm install...It's just the naming is confusing IMO because `WORKSPACE_NAME` AND `WORKSPACE_BUCKET` aren't `customEnvironmentVariables` in runtime routes. They're set by welder assuming some input from UI. 

If we call this just `environmentVariables`, then it makes more sense to me.

Anyways, now that you guys clarified this is for `WORKSPACE_NAME` AND `WORKSPACE_BUCKET` and potentially other envs, I'm fine with leaving it as is